src/types/core.ts

```diff
@@ -116,3 +120,11 @@ export interface ProviderDetails {
   url?: string;
   apiKey?: string;
 }
+
+export interface OcspResponse {
+  certificateId: string;
```

Comment:
I think it's better to split the different kind of response (successful / error) rather than merging.

Eventually the error one can extend the successfull one.

wdyt?

Reply:
like a OcspResponseRevoked and OcspResponseNotRevoked interfaces?

Reply:
yes